.github/workflows/frontend-integration-test.yml

```diff
@@ -27,37 +27,33 @@ jobs:
               - bin/**
               - catalogs/**
               - clients/client-java/**
-              - clients/client-java-runtime/**
-              - clients/filesystem-hadoop3/**
-              - clients/filesystem-hadoop3-runtime/**
               - common/**
               - conf/**
               - core/**
               - dev/**
               - gradle/**
-              - integration-test/**
+              - iceberg/**
+              - integration-test-common/**
               - meta/**
+              - scripts/**
               - server/**
               - server-common/**
-              - spark-connector/**
-              - trino-connector/**
               - web/**
-              - docs/open-api/**
               - build.gradle.kts
               - gradle.properties
               - gradlew
               - setting.gradle.kts
     outputs:
       source_changes: ${{ steps.filter.outputs.source_changes }}
 
-  # Integration test for AMD64 architecture
-  test-amd64-arch:
+  FrontendIT:
     needs: changes
     if: needs.changes.outputs.source_changes == 'true'
     runs-on: ubuntu-latest
     timeout-minutes: 60
     strategy:
       matrix:
+        # Integration test for AMD64 architecture
         architecture: [linux/amd64]
         java-version: [ 8 ]
```

Comment:
Some places we should java 8, some places we use 17, maybe we should unify them all.